package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "express": "^4.18.1"
+    "express": "^4.18.1",
+    "typescript": "^4.7.4"
```

Comment:
Should typescript be a dependency or a dev dependency?

Reply:
fixed in the last commit